### DISCUSSION TITLE
Domain is locked behind an account that is unaccessible, need help retreiving.

### DISCUSSION THREAD

**Discussion by bob-the-dev** [vercel/community#4089] [Help]
### Summary

Hi all, 

A while back I had a domain that was linked to a project I started under my previous employer's email address and github account. 
Since July I have left that company, and they also suspended / deleted my email address entirely and I also believe that they removed my github repo as well. 

Now, because I don't have access, there is no way for me to remove the domain settings of that project. 
I own the domain privately, but now I can not transfer the domain as I don't have access to the previous project.

Let me know if I am completely doomed.

### Steps to Reproduce

Not code related

**Comment by amyegan** [comment]
It sounds like you own the domain via a third party registrar. In that case, you can [add the custom domain](https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain) to your project and use a TXT record to [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access).
